CHANGELOG.md

```diff
@@ -17,6 +18,86 @@ The below sections record the changes for each API version in each release as fo
+  - OpenAPI [YAML spec file](https://github.com/camaraproject/WebRTC/blob/r2.1/code/API_definitions/webrtc-registration.yaml)
+
+### Added
+* Support registration expiry updates and termination event notifications at PR #83
+* Introduced `regSessionUpdate` schema with optional `registrationExpireTime` field to allow clients to request a new expiry time.
+* Added 200 OK response definition for `PUT /sessions/{registrationId}` to acknowledge successful updates.
+
+### Changed
+* fix: Updated deviceId format, examples and tests at #91
+* fix: registrationId became mandatory on registration response at #93
+* Commonalities aligment 0.6 at PR #88
+  * fix: Remove AUTHENTICATION_REQUIRED error code
+  * doc: Update `types` (multiple) property of `Subscription` schema
+  * feat: Error 422 MULTIEVENT_COMBINATION_TEMPORARILY_NOT_SUPPORTED added
+  * fix: Remove IDENTIFIER_MISMATCH error
+  * fix: x-correlator format update, along test and schema update 
+* Extended `regSessionResponse` schema to include `expiresAt`, indicating the assigned expiry time of the registration session at PR #83
+
+## webrtc-call-handling v0.3.0-rc.1
+
+- API definition **with inline documentation**:
+  - [View it on ReDoc](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/WebRTC/r2.1/code/API_definitions/webrtc-call-handling.yaml&nocors)
+  - [View it on Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/WebRTC/r2.1/code/API_definitions/webrtc-call-handling.yaml)
+  - OpenAPI [YAML spec file](https://github.com/camaraproject/WebRTC/blob/r2.1/code/API_definitions/webrtc-call-handling.yaml)
+
+### Changes
+* fix: Added 400 error response at #91
+* Commonalities aligment 0.6 at PR #88
+  * fix: Remove AUTHENTICATION_REQUIRED error code
+  * doc: Update `types` (multiple) property of `Subscription` schema
+  * feat: Error 422 MULTIEVENT_COMBINATION_TEMPORARILY_NOT_SUPPORTED added
+  * fix: Remove IDENTIFIER_MISMATCH error
+  * fix: x-correlator format update, along test and schema update
+
+## WebRTC Events v0.2.0-rc.1
+
+- API definition **with inline documentation**:
+  - [View it on ReDoc](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/WebRTC/r1.2/code/API_definitions/webrtc-events.yaml&nocors)
+  - [View it on Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/WebRTC/r1.2/code/API_definitions/webrtc-events.yaml)
+  - OpenAPI [YAML spec file](https://github.com/camaraproject/WebRTC/blob/r1.2/code/API_definitions/webrtc-events.yaml)
+
+### Added
+* Added new event type `webrtc-events:org.camaraproject.webrtc-events.v0.registration-ends` to notify subscribers when a registration ends at PR #83
+* Provided a concrete example for the `registration-ends` event subscription.
+
+### Changes
+* fix: Updated deviceId format, examples and tests at #91

```

Comment:
Seems that the links are not working in the resulting changelog in this format, propose to change them to the format which GitHub is creating within draft releases, then they will work also in local clones/downloads:

```suggestion
* fix: Updated deviceId format, examples and tests at https://github.com/camaraproject/WebRTC/pull/91
```

Please replace the other occurrences accordingly